letsencrypt/client.py

```diff
@@ -199,7 +199,7 @@ def obtain_certificate_from_csr(self, domains, csr,
         :param .le_util.CSR csr: DER-encoded Certificate Signing
             Request. The key used to generate this CSR can be different
             than `authkey`.
-        :param dict authzr: ACME Authorization Resource dict where keys are
+        :param list authzr: ACME Authorization Resource dict where keys are
```

Comment:
Looks like the description of `authzr` needs to be updated.

> ACME Authorization Resource dict where keys are domains and values are :class:`acme.messages.AuthorizationResource

isn't quite right.